extensions/wikia/ArticleVideo/templates/ArticleVideo_index.php

```diff
@@ -0,0 +1,23 @@
+<div id="article-video" class="article-video">
+	<img class="video-thumbnail" src="<?= $thumbnailUrl; ?>">
+	<div class="video-container">
+		<div class="video-placeholder">
+			<img class="video-thumbnail" src="<?= $thumbnailUrl; ?>">
+			<!-- TODO: replace with DS tiny icon -->
```

Comment:
Do we have a ticket for this TODO?
BTW HTML comments are visible in output HTML. To avoid that you could use PHP comments.